go/libkb/env.go

```diff
@@ -221,27 +221,38 @@ func (e *Env) GetUpdaterConfig() UpdaterConfigReader {
 }
 
 func (e *Env) GetMountDir() (string, error) {
+	var mountsubdir string
 	runMode := e.GetRunMode()
-	if runtime.GOOS == "windows" {
-		return e.GetString(
-			func() string { return e.cmd.GetMountDir() },
-			func() string { return os.Getenv("KEYBASE_MOUNTDIR") },
-			func() string { return e.GetConfig().GetMountDir() },
-		), nil
-	}
 	switch runMode {
 	case DevelRunMode:
-		return "/keybase.devel", nil
-
+		mountsubdir = "keybase.devel"
 	case StagingRunMode:
-		return "/keybase.staging", nil
-
+		mountsubdir = "keybase.staging"
 	case ProductionRunMode:
-		return "/keybase", nil
-
+		mountsubdir = "keybase"
 	default:
 		return "", fmt.Errorf("Invalid run mode: %s", runMode)
 	}
+
+	return e.GetString(
+		func() string { return e.cmd.GetMountDir() },
+		func() string { return os.Getenv("KEYBASE_MOUNTDIR") },
+		func() string { return e.GetConfig().GetMountDir() },
+		func() string {
+			switch runtime.GOOS {
+			case "darwin":
+				s, err := filepath.Abs(mountsubdir)
```

Comment:
Ah crap good catch.  Yeah I was trying to avoid that since I'm using `filepath` and thought I shouldn't use "/" directly.  But I'll try it with `filepath.Separator`.